### PR TITLE
fix(docs): update examples page

### PR DESCRIPTION
**Pull request by dhmlau**
Add missing examples to Example page: https://loopback.io/doc/en/lb4/Examples.html.
Since there are quite a lot of examples, so I did some categorization as well.

cc @raymondfeng @nabdelgadir 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by raymondfeng** [comment]
@dhmlau Good update. Pls fix the formatting issue at `examples/access-control-migration/README.md` with `npm run prettier:fix`.

File: docs/site/Examples.md [hunk 1]
```diff
@@ -6,35 +6,75 @@ sidebar: lb4_sidebar
...
   [`@loopback/context`](https://github.com/strongloop/loopback-next/tree/master/packages/context)
   as an Inversion of Control (IoC) and Dependency Injection (DI) container.
 
-- **[rpc-server](https://github.com/strongloop/loopback-next/tree/master/examples/rpc-server)**:
-  An example showing how to implement a made-up RPC protocol.
-
-- **[greeter-extension](https://github.com/strongloop/loopback-next/tree/master/examples/greeter-extension)**:
-  An example showing how to implement the extension point/extension pattern.
+- **[file-transfer](https://github.com/strongloop/loopback-next/tree/master/examples/file-transfer)**:
+  An example showing how to expose APIs to upload files.
```
Comment (jannyHou) [inline docs/site/Examples.md]:
```suggestion
  An example showing how to expose APIs to upload and download files.
```
?

File: docs/site/Examples.md [hunk 2]
```diff
@@ -6,35 +6,75 @@ sidebar: lb4_sidebar
...
+  and [Prometheus](https://prometheus.io/).
+
+### Examples with migration
+
+- **[lb3-application](https://github.com/strongloop/loopback-next/tree/master/examples/lb3-application)**:
+  An example demonstrating how to mount your existing LoopBack 3 application on
+  a new LoopBack 4 project.
+
+- **[access-control-migration](https://github.com/strongloop/loopback-next/blob/master/examples/access-control-migration)**:
+  An example demonstrating how to implement a Role Based Access Control(RBAC)
```
Comment (nabdelgadir) [inline docs/site/Examples.md]: Can you add a space between `Control(RBAC)`?

**Comment by raymondfeng** [comment]
@dhmlau Please split the `package-lock.json` changes into its own commit.

**Comment by dhmlau** [comment]
@slnode test please

**Comment by dhmlau** [comment]
@raymondfeng @derdeka, I've split the package locks changes from the Examples.md file change.

**Pull request merged**